.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+
+name: Build
+on:
+  push:
+    branches:
+      - master
+    tags-ignore:
+      - '*'
+    paths-ignore:
+      - README.adoc
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - README.adoc
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
This makes sure to fetch all tags as well, so that we can generate snapshot version correctly